services/scan/src/cli/retry-scan/index.test.ts

```diff
@@ -143,11 +143,9 @@ test('query with sheet ids', () => {
   const inputWorkspace = await createWorkspace(dirSync().name);
   const { store } = inputWorkspace;
 
-  store.setElection({
-    election: fixtures.election,
-    electionData: JSON.stringify(fixtures.election),
-    electionHash: '02f807b005e006da160b',
-  });
+  store.setElection(fixtures.electionDefinition);
+  store.setTestMode(false);
```

Comment:
Interestingly, this was needed because of the change in default from live to test mode in the scan service. It didn't fail in CI because this test doesn't run in CI due to resource constraints.

Reply:
huh, any chance we can run these more intensive tests in a non-blocking CI environment or something, maybe every now and then?

Reply:
/cc @carolinemodic to think about this issue, not blocking this PR of course.